modules/virt-about-workload-security.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * virt/virt-additional-security-privileges-controller-and-launcher.adoc
+
+:_content-type: CONCEPT
+[id="virt-about-workload-security_{context}"]
+= About workload security
+
+By default, virtual machine (VM) workloads do not run with root privileges in {VirtProductName}.
+
+For each VM, a `virt-launcher` pod runs an instance of `libvirt` to manage the VM process. The `virt-launcher` pod runs in _session mode_, meaning that all users in the `virt-launcher` container must have a nonzero user identifier (UID). Therefore, VMs run as unprivileged pods, adhering to the security principle of least privilege.
+
+There are no supported {VirtProductName} features that require root privileges. If a feature requires root, it might not be supported for use with {VirtProductName}.
+
+[NOTE]
+====
+link:https://virtio-fs.gitlab.io/[Virtiofs] is not supported because it is incompatible with session mode.
```

Comment:
Heads up, Kubevirt will implement this for 4.12+

Reply:
Is there an epic for this? If so, I want to make sure that we have a RN task and a task to remove this for 4.12.

Reply:
This will still be not supported but it will be possible to use. You can track https://issues.redhat.com/browse/CNV-15853?filter=-1